LPM-251.tex

```diff
@@ -49,6 +49,8 @@ \section{References\label{sect:references}}
 
 \section{Acronyms}
 \input{acronyms} % generated by the acronyms.csh (GaiaTools)
+
+\input{checklist}
```

Comment:
```suggestion
\section{IDAC Proposal Checklist}\label{sec:checklist}
\input{checklist}
```

Reply:
For consistency with how acronyms are handled?

Reply:
acronyms is generated this is not ..